selfdrive/manager.py

```diff
@@ -204,6 +204,7 @@ def get_running():
   'radard',
   'dmonitoringd',
   'calibrationd',
+  'dmonitoringmodeld',
```

Comment:
Only with WEBCAM flag. Make sure it's not added twice. It's also in the ANDROID section.